discovery/seata-discovery-zk/src/test/java/io/seata/discovery/registry/zk/ZookeeperRegistryProviderTest.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.discovery.registry.zk;
+
+import io.seata.discovery.registry.RegistryService;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Geng Zhang
+ */
+public class ZookeeperRegistryProviderTest {
+
+    @Test
+    public void provide() {
+        ZookeeperRegistryProvider provider = new ZookeeperRegistryProvider();
+        RegistryService registryService = provider.provide();
```

Comment:
and this line can be removed too.
the Assert.assertTrue(registryService instanceof ZookeeperRegisterServiceImpl); 
can be replaced with the following code:
Assert.assertTrue(provider.provide() instanceof ZookeeperRegisterServiceImpl);

Reply:
This line is for testing the return singleton `ZookeeperRegisterServiceImpl `.